### ISSUE TITLE
Setting up traccar online

### ISSUE THREAD

**Issue by tomasrage** [tananaev/traccar#3569]
Hello guys, yesterday I start with Traccar, just setup one GPS to my windows 10 computer and working charm... now I have a little issue and I don't know how or where to start... I wanna track the gps from my celular or from another computer, I try to open in another computer the static ip of my windows 10 computer but can't get in... what I have to do? How I can turn my computer with windows 10 and static ip into a server or something like that to acces traccar over internet? thanks in advance for the time

**Comment by tananaev** [comment]
You should be able to access Traccar server remotely. If it doesn't work, most likely you have some network issue (e.g. router, firewall config).

**Comment by tomasrage** [comment]
Rigth just open port in router and works now i cant conect the ios app on startup ask for server input ip and conection failed... need another port for traccar manager ios app?

**Comment by tananaev** [comment]
What URL are you trying?

**Comment by tomasrage** [comment]
Http://189.208.28.76:8082 works on web not working on ios app

**Comment by tananaev** [comment]
It should work fine.

**Comment by tomasrage** [comment]
Yes now is working fine i did nothing just a few moment after i wrote this post start working the ios app

Thank you

Enviado desde mi iPhone

El 06/10/2017, a las 8:31 a.m., Anton Tananaev <notifications@github.com<mailto:notifications@github.com>> escribió:


It should work fine.
